tracks/bash/exercises/two-fer/mentoring.md

```diff
@@ -0,0 +1,79 @@
+### Reasonable Solutions
+
+```
+#!/bin/bash
```

Comment:
I think `#!/usr/bin/env bash` is a better shebang than `#!/bin/bash` because of [portability](https://stackoverflow.com/questions/10376206/what-is-the-preferred-bash-shebang) reason. The former is also included in the template in Hello World exercise.

Reply:
Changed it.